### ISSUE TITLE
Can not add employee into the work shift.

### ISSUE THREAD

**Issue by burakbozyigit** [orangehrm/orangehrm#1857]
### OrangeHRM version

5.6.1

### Environment details

- OrangeHRM source: Namecheap Hosting's Softaculous Installer
- Platform: Linux 4.18.0-513.11.1.lve.el8.x86_64
- PHP version: PHP Version 8.0.30
- Database and version: 10.6.18-MariaDB-cll-lve
- Web server: LiteSpeed V8.1
- Browser: Google Chrome 125.0.6422.142 on Mac OS Sonoma 14.5

### Describe the bug

When I wanted to add an employee to the shift, it throws an error.

<img width="418" alt="Screenshot 2024-06-20 at 15 42 00" src="https://github.com/orangehrm/orangehrm/assets/3204320/c69e9ec2-b7be-415d-b006-e48084003422">

<img width="1257" alt="Screenshot 2024-06-20 at 15 47 04" src="https://github.com/orangehrm/orangehrm/assets/3204320/35a68060-b5c3-40dc-912b-7eb133708f5c">

### To reproduce

Admin > Job > Work Shifts > Edit Work Shift > Tried to type down Employee names. 
It throws a red error bubble says "Unexpected error" with a log.

### Expected behavior

Add that worker into the shift schedule.

### Relevant log output

```shell
[2024-06-20T12:30:42.061828+00:00] orangehrm.CRITICAL:
#0 /home/HOMEDIR/src/plugins/orangehrmCorePlugin/Api/V2/Serializer/NormalizerTrait.php(131): OrangeHRM\Pim\Service\Model\EmployeeModel->__construct(Array)
#1 /home/HOMEDIR/src/plugins/orangehrmCorePlugin/Api/V2/Serializer/NormalizerTrait.php(100): OrangeHRM\Core\Api\V2\Serializer\AbstractEndpointResult->getInitializedModelInstance(Array)
#2 /home/HOMEDIR/src/plugins/orangehrmCorePlugin/Api/V2/EndpointCollectionResult.php(36): OrangeHRM\Core\Api\V2\Serializer\AbstractEndpointResult->normalizeObjectsArray()
#3 /home/HOMEDIR/src/plugins/orangehrmCorePlugin/Controller/Rest/V2/GenericRestController.php(207): OrangeHRM\Core\Api\V2\EndpointCollectionResult->normalize()
#4 /home/HOMEDIR/src/plugins/orangehrmCorePlugin/Controller/Rest/V2/GenericRestController.php(87): OrangeHRM\Core\Controller\Rest\V2\GenericRestController->getPreparedResponseParamsFromResult(Object(OrangeHRM\Core\Api\V2\EndpointCollectionResult))
#5 /home/HOMEDIR/src/plugins/orangehrmCorePlugin/Controller/Rest/V2/AbstractRestController.php(145): OrangeHRM\Core\Controller\Rest\V2\GenericRestController->handleGetRequest(Object(OrangeHRM\Core\Api\V2\Request))
#6 /home/HOMEDIR/src/vendor/symfony/http-kernel/HttpKernel.php(163): OrangeHRM\Core\Controller\Rest\V2\AbstractRestController->handle(Object(OrangeHRM\Framework\Http\Request))
#7 /home/HOMEDIR/src/vendor/symfony/http-kernel/HttpKernel.php(75): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(OrangeHRM\Framework\Http\Request), 1)
#8 /home/HOMEDIR/src/lib/framework/Framework.php(156): Symfony\Component\HttpKernel\HttpKernel->handle(Object(OrangeHRM\Framework\Http\Request), 1, true)
#9 /home/HOMEDIR/web/index.php(41): OrangeHRM\Framework\Framework->handleRequest(Object(OrangeHRM\Framework\Http\Request))
#10 {main} [] []
```

**Comment by devishke-orange** [comment]
Hi @burakbozyigit,

We have fixed this in OrangeHRM 5.7 which will be releasing soon. You can take a look at the fix here: https://github.com/orangehrm/orangehrm/pull/1846

**Comment by burakbozyigit** [comment]
Thank you so much for the link and resolution!
https://github.com/orangehrm/orangehrm/pull/1846
